### DISCUSSION TITLE
挂载阿里云盘的时候里面的信息显示不全，而且里面的文件全部

### DISCUSSION THREAD

**Discussion by qiiaenn** [AlistGo/alist#6975] [Q&A]
![image](https://github.com/user-attachments/assets/1441d1b1-0a62-4082-8116-423726658055)
**这个是list文件存储的地方**
![image](https://github.com/user-attachments/assets/bc275753-d5e0-4b8a-ab24-92bfc291356e)
**这个是我阿里云盘的存储信息**
![image](https://github.com/user-attachments/assets/6df1d03d-3d26-4db8-b9d4-0bec121e0f60)
![image](https://github.com/user-attachments/assets/0926dd8e-92aa-4e5e-8752-3e220a8634bf)
![image](https://github.com/user-attachments/assets/7aa06c1c-87b7-4f31-ba81-733d1e7e6da0)
**这个是添加云盘的后台**

**Comment by anwen-anyi** [comment]
你应该是后来改动的配置吧，改动完配置保存后使用右下角刷新一下，而且你选择的 `云盘类型` 是备份盘，你应该选择资源库，就能看到除了 `备份文件` 之外全部文件
![image](https://github.com/user-attachments/assets/3c098cfc-5bf5-4d82-8530-37e372dc655e)
